services/auth-app/README.md

```diff
@@ -0,0 +1,30 @@
+# Auth-App
+
+The auth-app service provides authentication for 3rd party apps.
+
+## The `auth` Service Family
+
+ocis uses serveral authentication services for different use cases. All services that start with `auth-` are part of the authentication service family. Each member authenticates requests with different scopes. As of now, these services exist:
+  -   `auth-basic` handles basic authentication
+  -   `auth-bearer` handles oidc authentication
+  -   `auth-machine` handles interservice authentication when a user is impersonated
+  -   `auth-service` handles interservice authentication when using service accounts
+  -   `auth-app` handles authentication of external 3rd party apps
```

Comment:
Note, you need to provide the new service also to the readme's of the other services...
They _should_ / _must_ have a identical list...

Reply:
That's what I did. See `Readme.md` files in `auth-bearer`, `auth-machine` and `auth-service`